tools/scripts/storage/init_db.sql

```diff
@@ -107,6 +106,16 @@ CREATE TABLE IF NOT EXISTS `task_instances`
     PRIMARY KEY (`id`)
 );
 
+CREATE TABLE IF NOT EXISTS `scheduler_leases`
+(
+    `scheduler_id` BINARY(16) NOT NULL,
+    `task_id`      BINARY(16) NOT NULL,
+    `lease_time`   TIMESTAMP  NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    CONSTRAINT `lease_scheduler_id` FOREIGN KEY (`scheduler_id`) REFERENCES `schedulers` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE,
+    CONSTRAINT `lease_task_id` FOREIGN KEY (`task_id`) REFERENCES `tasks` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE,
+    INDEX (`scheduler_id`)
+);
+
```

Comment:
Is this change unrelated to the PR? There is no mention of it in the description and none of the other changes are related to leases or require a new table (as far as I can tell).

Is it possible to move this to another more relevant PR?

Reply:
After looking at #98 I'm pretty sure this table should be removed from this PR and added in that one.
```suggestion
```